crates/proto/src/rr/rdata/caa.rs

```diff
@@ -8,7 +8,10 @@
 //! allows a DNS domain name holder to specify one or more Certification
 //! Authorities (CAs) authorized to issue certificates for that domain.
 //!
-//! [RFC 6844, DNS Certification Authority Authorization, January 2013](https://tools.ietf.org/html/rfc6844)
+//! Two major RFC:
+//!
```

Comment:
I don't think we need to reference the old RFC here, just reference the new one (it refers to the older one).

Reply:
Sure will make a fix tonight or tomorrow moring. Thanks :)